js_modules/dagit/packages/core/src/assets/AssetEvents.tsx

```diff
@@ -89,6 +101,48 @@ function useRecentAssetEvents(
   }, [data, loading, refetch, loadUsingPartitionKeys]);
 }
 
+function useRecentRunWarnings(
+  assetKey: AssetKey,
+  grouped: AssetEventGroup[],
+  repositorySelector?: RepositorySelector,
+) {
+  const {data} = useQuery<LastRunQuery, LastRunQueryVariables>(LAST_RUNS_QUERY, {
+    skip: !repositorySelector,
+    variables: {
+      repositorySelector: repositorySelector!,
+    },
+  });
+  return React.useMemo(() => {
+    const lastRunInfo =
+      data?.repositoryOrError.__typename === 'Repository' &&
+      data.repositoryOrError.latestRunByStep.length > 0
+        ? data.repositoryOrError
+        : null;
+    const latestRuns: LastRunQuery_repositoryOrError_Repository_latestRunByStep_LatestRun[] = [];
+    const jobRunsCounts: LastRunQuery_repositoryOrError_Repository_latestRunByStep_JobRunsCount[] = [];
+    for (const item of lastRunInfo?.latestRunByStep || []) {
+      if (item.__typename === 'LatestRun') {
+        latestRuns.push(item);
+      } else if (item.__typename === 'JobRunsCount') {
+        jobRunsCounts.push(item);
+      }
+    }
+
+    const assetName = assetKey.path[assetKey.path.length - 1];
```

Comment:
Ahh crap can we make this `assetNode.opName` i think they were talking yesterday about making this mapping different where it'd be something like a__b for `["a", "b"]`